### ISSUE TITLE
Unable to log on (Solution)- Signups Disabled by default

### ISSUE THREAD

**Issue by cheeseyfish** [0perationPrivacy/VoIP#55]
**Platform:** Both Desktop and Mobile
**Browser:** Firefox and Brave
**App Version:** v0.66-beta

### Describe the bug
Hello, I followed the guide as stated at https://inteltechniques.com/sms.html
I created the Github Fork, went to my webapp and I am up to the lines that say: 
"Click "Sign Up"
Log into your new account
Click the dropdown menu and select "Add New Profile""
The issue is that I cannot log onto the account. I verified I am using the correct username and password. There is no error message. It acts as though I never clicked or touched the "Login" button.

**To Reproduce**
As stated. This also occurs at https://voip.operationprivacy.com/ for me. Being on or off a VPN did not change the outcome. 

Thanks for your work.

**Comment by privacyosint** [comment]
Go to the logs and put them in here. 
Go to deloy and hit Deploy branch.
Click View build log.

Send me those logs.
You can email me or send them here. 
osint.minidisk@8shield.net

**Comment by cheeseyfish** [comment]
@privacyosint 
I searched around and found this comment https://github.com/0perationPrivacy/VoIP/issues/51#issuecomment-933715236 and didn't understand what was meant. But then I found this in the instructions:

Return to https://dashboard.heroku.com/apps
Select your app
Click "Settings"
Click "Reveal Config Vars"
In the "KEY" field enter "SIGNUPS"
In the "VALUE" field enter "off"
Click "Add"

I followed this, turned SIGNUPS to "on" and it worked.
Thanks for the help!

**Comment by Nickfoxtel** [comment]
Hi Cheeseyfish,

I'm having the same problem as you were.  Last year, I was able to successfully setup and SignUp for the VoIP application and had it working with a phone number.  I have since released that number and decided to start the process all over again with a fresh one.  The only account I kept the same was my Heroku account.  And I had deleted the previous app there.  

So I just started anew with the MongoDB acct. & Github creation processes, and returned to Heroku to set up a new app there.  I got to the problem you indicated above, wherein a "404" symbol just pops up on attempts to Sign Up.  
I tried adding ```SIGNUPS``` to the ```KEY``` field, and ```on``` to the ```VALUE``` field, then clicked the "Add" button.  However, it didn't work, & I could only get the "404" error to pop up on screen after attempting SignUp.

Was there anything else you tried?  I made sure to enter the password correctly (no special characters) in the db string, & yes, without the <> brackets.
